index.html

```diff
@@ -1086,9 +1086,25 @@
 
 ## Principles for Vulnerable People
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Some people, or groups of people, are at greater risk from the asymmetries of power created by the
+concentration of information power ([[?Automating-Inequality]]). [=People=] who for whatever reason
+are in a more [=vulnerable=] position with respect to data [=processing=] should be more proactively
+protected by the [=user agent=] as well as by [=parties=].
+
+One issue, however, is that signalling to [=parties=] that a [=person=] may be [=vulnerable=] is
+itself a sensitive piece of information that should not be shared. One way to approach this issue is
+to rely on a [=global opt-out=] mechanism such as [[?GPC]] that is used by [=people=] whether they
+are [=vulnerable=] or not and puts a requirement on [=parties=] to minimise their [=data=]
+[=processing=].
```

Comment:
I think parties already have a requirement to minimise their data processing ... Maybe just remove this clause:

```suggestion
are [=vulnerable=] or not.
```

The point here is to let the vulnerable people blend into a larger population, not to re-describe what GPC in particular does.